sequencing/vcf_summary/dxapp.json

```diff
@@ -3,34 +3,42 @@
   "title": "VCF Summary Statistics",
   "summary": "Generate summary statistics for a VCF file (by sample and by variant)",
   "dxapi": "1.0.0",
-  "version": "0.0.1",
+  "version": "0.1.0",
```

Comment:
I think we're setting the versions to 1.0.0, not 0.1.0. (I had it wrong initially.)